_vector-search/ai-search/hybrid-search/collapse.md

```diff
@@ -0,0 +1,252 @@
+
+The following example is taken from the documentation for [collapse in non-hybrid query](https://docs.opensearch.org/docs/latest/search-plugins/collapse-search/) and modified for hybrid query:
+
+
+Create an index:
+```json
+PUT /bakery-items
+{
+  "mappings": {
+    "properties": {
+      "item": {
+        "type": "keyword"
+      },
+      "category": {
+        "type": "keyword"
+      },
+      "price": {
+        "type": "float"
+      },
+      "baked_date": {
+        "type": "date"
+      }
+    }
+  }
+}
+```
+
+Index documents:
+```json
+POST /bakery-items/_bulk
+{ "index": {} }
+{ "item": "Chocolate Cake", "category": "cakes", "price": 15, "baked_date": "2023-07-01T00:00:00Z" }
+{ "index": {} }
+{ "item": "Chocolate Cake", "category": "cakes", "price": 18, "baked_date": "2023-07-04T00:00:00Z" }
+{ "index": {} }
+{ "item": "Vanilla Cake", "category": "cakes", "price": 12, "baked_date": "2023-07-02T00:00:00Z" }
+```
+
+Create a search pipeline:
+```json
+PUT /_search/pipeline/norm-pipeline
+{
+  "description": "Normalization processor for hybrid search",
+  "phase_results_processors": [
+    {
+      "normalization-processor": {
+        "normalization": {
+          "technique": "min_max"
+        },
+        "combination": {
+          "technique": "arithmetic_mean"
+        }
+      }
+    }
+  ]
+}
+```
+
+Search the index, collapsing on the `item` field:
+```json
+GET /bakery-items/_search?search_pipeline=norm-pipeline
+{
+  "query": {
+    "hybrid": {
+      "queries": [
+        {
+          "match": {
+            "item": "Chocolate Cake"
+          }
+        },
+        {
+          "bool": {
+            "must": {
+              "match": {
+                "category": "cakes"
+              }
+            }
+          }
+        }
+      ]
+    }
+  },
+  "collapse": {
+    "field": "item"
+  }
+}
+```
+
+Response:
+```json
+{
+  "took": 2,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 3,
+      "relation": "eq"
+    },
+    "max_score": 1.0,
+    "hits": [
+      {
+        "_index": "bakery-items",
+        "_id": "KS-5YJcBeeOHd7EbgdPT",
+        "_score": 1.0,
+        "_source": {
+          "item": "Chocolate Cake",
+          "category": "cakes",
+          "price": 15,
+          "baked_date": "2023-07-01T00:00:00Z"
+        },
+        "fields": {
+          "item": [
+            "Chocolate Cake"
+          ]
+        }
+      },
+      {
+        "_index": "bakery-items",
+        "_id": "Ky-5YJcBeeOHd7EbgdPT",
+        "_score": 0.5005,
+        "_source": {
+          "item": "Vanilla Cake",
+          "category": "cakes",
+          "price": 12,
+          "baked_date": "2023-07-02T00:00:00Z"
+        },
+        "fields": {
+          "item": [
+            "Vanilla Cake"
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+You can also use sort in addition to collapsing the documents:
```

Comment:
this section needs more context: why only sort, how about other features like pagination or explain etc. Is this combination of sort and collapse work differently comparing to other queries?

Reply:
No, it works the exact same.  Should I just remove this section?